generator.yaml

```diff
@@ -59,6 +61,11 @@ resources:
         from:
           operation: DescribeCertificate
           path: Certificate.CreatedAt
+      DomainValidations:
+        is_read_only: true
+        from:
+          operation: DescribeCertificate
+          path: Certificate.DomainValidationOptions
```

Comment:
Note: we need to extend the code generator to automatically set this fields on Create/ReadOne using something like:
```yaml
        set:
        - from: Certificate.DomainValidationOptions
          method: ReadOne
```

Reply:
Opened https://github.com/aws-controllers-k8s/community/issues/2010